mifosng-android/src/main/java/com/mifos/mifosxdroid/online/savingsaccount/SavingsAccountFragment.java

```diff
@@ -299,24 +306,19 @@ public void onDestroyView() {
     @Override
     public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
         switch (parent.getId()) {
-            case R.id.sp_interest_comp:
-                interestCompoundingPeriodTypeId = mSavingProductsTemplate
-                        .getInterestCompoundingPeriodTypeOptions().get(position).getId();
-                break;
-            case R.id.sp_days_in_year:
-                interestCalculationDaysInYearTypeId = mSavingProductsTemplate
-                        .getInterestCalculationDaysInYearTypeOptions().get(position).getId();
-                break;
-            case R.id.sp_interest_calc:
-                interestCalculationTypeAdapterId = mSavingProductsTemplate
-                        .getInterestCalculationTypeOptions().get(position).getId();
-                break;
-            case R.id.sp_interest_p_period:
-                interestPostingPeriodTypeId = mSavingProductsTemplate
-                        .getInterestPostingPeriodTypeOptions().get(position).getId();
+            case R.id.sp_field_officer:
+                fieldOfficerId = mSavingProductsTemplateByProductId
+                        .getFieldOfficerOptions().get(position).getId();
                 break;
             case R.id.sp_product:
                 productId = mProductSavings.get(position).getId();
+                if (clientId != 0) {
```

Comment:
Do not use ```clientId```,  use ```isGroupAccount```. it will give you what you are going to create a GroupSavings or ClientSavings Account.